apps/heft/LICENSE

```diff
@@ -0,0 +1,24 @@
+@rushstack/heft
```

Comment:
We shouldn't publish Heft without a README.md file. At the very least, I'd suggest to start with a placeholder file with a notice saying something simple like "This project is still under development and is being published to facilitate testing. For more details and support resources, please visit: https://rushjs.io"

Reply:
It's currently not set to publish, but I can add a simple README for now.